package.json

```diff
@@ -225,6 +225,43 @@
           "markdownDescription": "Enable/disable ability to format document. Default is `true`.",
           "scope": "window"
         },
+        "xml.format.experimental": {
+          "type": "boolean",
+          "default": false,
+          "markdownDescription": "Enable/disable experimental formatter. Default is `false`.",
+          "scope": "window"
+        },
+        "xml.format.maxLineWidth": {
+          "type": "integer",
+          "default": 80,
+          "markdownDescription": "Max line width. Default is `80`.",
+          "scope": "window"
+        },
+        "xml.format.grammarAwareFormatting": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Use Schema/DTD grammar information while formatting. Default is `true`.",
+          "scope": "window"
+        },
+        "xml.preferences.preserveSpace": {
```

Comment:
It should be better to have `xml.formatpreserveSpace` but technically XMLFormattingOptions extends LSP4J FormattingOptions which cannot support array of string (only boolean, number and string). It is the reason why I set this settings in the xml.preferences.

Reply:
Ok I fixed the problem on LemMinX side, now we have `xml.format.preserveSpace`